src/renderer/containers/map/infoBox/index.jsx

```diff
@@ -32,13 +32,20 @@ export default class InfoBox extends React.Component {
     const address = `https://www.google.com/maps/place/${conversion(
       details.address1
     )}+${details.zipCode}`;
+    const email = {
+      to: 'smartcolumbusos@columbus.gov',
+      subject: 'Service Location No Longer Active',
+      body: `The following service area is no longer active.\n\n\t${
+        details.name
+      }\n\t${details.address1}\n\t${details.address2}\n\tPhone: ${
+        details.areaCode
+      }${details.phoneNumber}`,
+    };
     return (
```

Comment:
this is a minor point, no need to change if you wish, although I vote to add the Dataset link in body of the email. This way atleast the team will know which dataset has issue for which the report is received.
Dataset - https://discovery.smartcolumbusos.com/dataset/handson_central_ohio/570a8e02_fb0e_4cee_895b_3b32bd740650
Organization - HandsOn Central Ohio

Reply:
lets create a card for this. would like to make the dataset a link but not really sure how to do that right now. I think we can add that ability to make it a link in another card.

Reply:
card created: https://github.com/orgs/SCODEMeetup/projects/8#card-25930237